src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -237,22 +240,63 @@
             <argument type="service" id="sylius.repository.payment_method" />
         </service>
 
-        <service id="sylius.taxation.order_shipment_taxes_by_zone_applicator" class="%sylius.taxation.order_shipment_taxes_by_zone_applicator.class%">
+        <service id="sylius.taxation.order_shipment_taxes_applicator" class="%sylius.taxation.order_shipment_taxes_applicator.class%">
```

Comment:
I'm for keeping it simple and since we don't plan to have any other types of applicators in core I think it's fine to leave it as it is now :)